validation/string_end.go

```diff
@@ -0,0 +1,36 @@
+package validation
+
+import (
+	"strings"
+)
+
+// DoesntEndWithValidator validates that the string under validation doesn't end with the given suffix.
+type DoesntEndWithValidator struct {
+	BaseValidator
+	Suffix string
+}
+
+// Validate checks if the field under validation satisfies this validator's criteria.
+func (v *DoesntEndWithValidator) Validate(ctx *Context) bool {
+	str, ok := ctx.Value.(string)
+	if !ok {
+		return false
+	}
+
+	return !strings.HasSuffix(str, v.Suffix)
```

Comment:
To make it consistent with other prefix/suffix validators, `v.Suffix` should be a `[]string` and the validator should check that the value doesn't end with **any** of the suffixes.